cmd/lakectl/cmd/root.go

```diff
@@ -60,6 +61,12 @@ type Configuration struct {
 	} `mapstructure:"credentials"`
 	Server struct {
 		EndpointURL lakefsconfig.OnlyString `mapstructure:"endpoint_url"`
+		Retries     struct {
+			Enabled         bool          `mapstructure:"enabled"`
+			MaxRetries      int           `mapstructure:"max_retries"`
+			MinWaitInterval time.Duration `mapstructure:"min_wait_interval"`
+			MaxWaitInterval time.Duration `mapstructure:"max_wait_interval"`
```

Comment:
Can you please document this two

Reply:
Added inline comments. If you can point me to the documentation, I'll add it there.

Reply:
I guess a section [here](https://docs.lakefs.io/reference/cli.html#lakectl-lakefs-command-line-tool) mentioning retries would be appropriate..